src/api/app/models/token.rb

```diff
@@ -7,6 +7,11 @@ class Token < ApplicationRecord
 
   has_secure_token :string
 
+  before_validation do
+    self.name = '' if name.nil?
+  end
+
+  validates :name, length: { maximum: 30 }
```

Comment:
This validation doesn't match the length of the column anymore.

```suggestion
  validates :name, length: { maximum: 64 }
```

Reply:
Ops. Changed.